src/main/java/com/trailfinder/service/HikeServiceStub.java

```diff
@@ -23,31 +23,14 @@
 	public ProfileDTO getProfile(int profileId) {
 		// TODO create a List of profiles and check if the passed Id matches any
 		List<ProfileDTO> profiles = new ArrayList<ProfileDTO>();
-		// Create 5 profiles
-		int i;
-		for (i = 0; i < 5; i++)
-		{
-			ProfileDTO testProfile = new ProfileDTO();
-			testProfile.setProfileId(i);
-			testProfile.setFirstName("TestName" + i);
-			testProfile.setBio("TestBio");
-			testProfile.setEvents(null);
-			profiles.add(testProfile);
-		}
 		
-		// Check which profile was selected
-		ProfileDTO profile = new ProfileDTO();
-		switch (profileId) 
+		// Create 5 profiles
+		for (int i = 0; i < 5; i++)
 		{
-			case 0: profile = profiles.get(0); break;
-			case 1: profile = profiles.get(1); break;
-			case 2: profile = profiles.get(2); break;
-			case 3: profile = profiles.get(3); break;
-			case 4: profile = profiles.get(4); break;
-			default: profile = null; break;
+			profiles.add(new ProfileDTO(i, "TestName" + i, "TestBio", null));
 		}
 		
-		return profile;
+		return profileId >= 0 && profileId < profiles.size() ? profiles.get(profileId) : null;
```

Comment:
Removed switch statement so that profileId could be checked within range. This way you could change the amount of Profiles you generate and test them without having to add another case to the switch statement